CHANGELOG.md

```diff
@@ -1,5 +1,243 @@
 # Changelog
 
+## v0.66.0
+
+## v0.66.0-rc.0
+
+### Breaking
+
+- Bump Hermes npm to 0.8.0 ([2b67631210](https://github.com/facebook/react-native/commit/2b67631210070de0c07b21c69fb9d0ccd9e9fcba) by [@Huxpro](https://github.com/Huxpro))
+
```

Comment:
The commits behind the removals of the deprecated components (`Picker`, `PickerIOS` and `StatusBarIOS`) should be a part a of the breaking changes too.